### DISCUSSION TITLE
Megane 3 - EOL Settings after reset configuration

### DISCUSSION THREAD

**Discussion by wotekPL** [cedricp/ddt4all#1256] [Q&A]
Hi, 

i reset the configuration on my Rlink in Megane 3 and now i get message "Please Add Settings" so can i repair without clip and reprog or  clip is the last chance?

**Comment by AndyIreland** [comment]
did you press the soft reset or hard reset button?
Is it rlink 1 or rlink2 in your Megane?

**Comment by wotekPL** [comment]
I press reset configuration

**Comment by AndyIreland** [comment]
is it rlink1 tomtom or is it rlink2 rlink fitted to your megane ?

**Comment by wotekPL** [comment]
OMG :D Sorry. It's a Megane 3 with Rlink 1

**Comment by AndyIreland** [comment]
ah right thanks. I have repaired my rlink1 in my renault zoe once when it had EOL triangle before with DDT4ALL by filling in settings. 
Its a bit difficult to explain , but if you have a elm327 bluetooth OBD 2 and the specially wired cable to read the rlink through the OBD2 port and a windows laptop and can get an internet signal on your laptop inside your car I can log on over the internet and have a look remotely from my end to see what settings are filled in and see if i can see other settings missing if you like, if you want me to just let me know.

**Comment by wotekPL** [comment]
Yes Sure! 
Now cannot turn on radio and navigation :( 

I have clip or ddt4all. 

Do you think we can do anything else?

If i change bic to old version screen can turn on but with information abaout settings. If i change bic to newer version then i cannot turn on and if i click eject disk then led blink 10x.

**Comment by wotekPL** [comment]
Ok. Actually info about EOL is back.

**Comment by wotekPL** [comment]
@AndyIreland how can i launch firmware from your post. I format sd card in FAT32 but software not launched?
I copy file tomtom.000 but canot launch this.

**Comment by wotekPL** [comment]
@cedricp, @shrlnm 
![Screenshot_20220913_160128_org quark dr canapp](https://user-images.githubusercontent.com/11139267/190382649-de129557-1e7b-4775-9c69-4e34070d51ac.jpg)

Software Number is a 333C and ddt4all cannot autoidenty version MFD so how can i overwrite this row?

**Comment by AndyIreland** [comment]
OK , so on my Renault Zoe I have a 'Home' button next to the touch screen itself - i dont know how the Megane 3 is configured with its buttons but i pressed the Home button on mine with 5 pushes with the tomtom.000 file on the card inserted and it started to load the firmware. My Zoe does not have a BIC unit in it like yours so you will have to make sure that is configured as well I think because sometimes when you press the 'reset software' in the mfd settings it can erase settings in BIC as well I think. 

other things you can try is that if the MFD does not auto find in DDT4ALL you have to set it manually choosing MFD 5.4 from the list then choose your settings then press home button 5 times or 'hard reset' within DDT4ALL to reboot the rlink

Another thing you could try is formatting a 8gb USB stick to Fat32 copy over the tomtom.000 file to it and try putting that into usb socket of car and see if that starts loading the software

**Comment by wotekPL** [comment]
Ok could you show me screen with all data?

What can i generate VechicleIdentifier im BIC configuration?

Screen here https://github.com/cedricp/ddt4all/issues/92#issuecomment-457928308

**Comment by wotekPL** [comment]
Yes, i try with pendrive format to FAT32 and new  card with format. Maybe i format card with map and then copy tomtom.000 to card? What do you think?

**Comment by AndyIreland** [comment]
No, i would just copy the TomTom file to fresh SD card not the map card. 

Try use this tomtom.000 file its 11.346 version
https://we.tl/t-GCzQGyQoK3

**Comment by wotekPL** [comment]
Eh, unfortunately but i put card or USB stick and nothing happend.

**Comment by AndyIreland** [comment]
> Eh, unfortunately but i put card or USB stick and nothing happend.

ah right it must be because the radio is in EOL mode then it cannot boot SD card.

i cannot really think of anything else  then apart from manually selecting MFD 5.4 in DDT4ALL and filling out all the parameters that you can and see if that works - hard to tell what is going on without seeing for myself what is filled out and what is not , and I can only really look if I can log on remotely over the internet.

**Comment by wotekPL** [comment]
I bought clip but also i have a problem. 
I cannot autoconfiguration because clip cannot this file on reprog or if i reprog rLinka by RUSB. So maybe somone knows what od RUSB on can clip :(

**Comment by shrlnm** [comment]
If you  did reset configuration, then I may generate parameters for your VIN and auto configuration macro for pyren. I need your VIN and screenshot of ID screen from pyren.
Also I have to notice that  there are complaints that pyren can't write long commands to r-link devices. It is related to timeout parameters. It is probably will require to tune a code a bit for  implementing auto-configuration macro on rlink1.

**Comment by wotekPL** [comment]
@shrlnm thank you!
Screenshot from pyren. 

Navi
X

BIC
X
or i have it this scren from ddt

X

**Comment by AndyIreland** [comment]
@shrlnm if i were to supply a dump text file (.txt) of my 2012 radio file or dump of mfd5.4 txt dump file and sent it to someone with EOL and just changed the VIN section to their VIN and CRC then do a Hard Reset would that get their EOL disappeared andtheir r-link 1 working again?

if it did, what PIN code would they need to put in when starting up if it asked for it , would it be mine or theirs?

**Comment by shrlnm** [comment]
@AndyIreland unfortunately I don't know answers on your questions as so I have no experience with r-link1.
@wotekPL Here are commented configurations for your VIN
Implement on your own risk
You may try to implement them as macros in pyren or just as guide with  your parameters values 

(links was deleted) look in your mail box

**Comment by wotekPL** [comment]
@shrlnm 

line with $addr should be empty? 

`$addr = `

**Comment by shrlnm** [comment]
@wotekPL it is a mistake. Insert an address here

**Comment by wotekPL** [comment]
Which address can i write?

**Comment by shrlnm** [comment]
TargetAddress from Uces.xml  for your ECU

    <EcuData name="11217" doc="MFD_EUR_X38PH2_0000_14_B" stdType="STD_B">
        <SessionId>genSession:GenSession?GenSession#11217</SessionId>
        <TrtId>diagServices:EcuDiag?EcuDiag#11217</TrtId>
        <ModelId>FG1011217.XML</ModelId>
        <OptimizerId>SG1011217.XML</OptimizerId>
        <ECUInformations>
            <Address **targetAddress="57"** toolAddress="F1" addressExtension="00" MType="0" TAType="0"/>

**Comment by wotekPL** [comment]
@shrlnm - how can i thank you? You're made my day! 

T H A N K  Y O U ! ! !

Rlink is back!

**Comment by shrlnm** [comment]
Ok, glad to hear it. )
Could you tell me, which of the macro helped?

**Comment by wotekPL** [comment]
@shrlnm cmd_10_4461_259205564R.cmd 
After use it i saw rlink logo.

**Comment by shrlnm** [comment]
@wotekPL Thank you for feedback. One more question. Did you use USB-ELM or a BT/WIFI? If you saved a log could you share it? Your information may help others to repair their rlinks.
Thanks in advance

**Comment by wotekPL** [comment]
I used BT-ELM (ELM327 iCar2 Vgate)

[elm_log.txt]
[ecu_log.txt]

@shrlnm - Thank you - again :1st_place_medal:

**Comment by shrlnm** [comment]
Thank you. I've downloaded them. You may remove them from here.

**Comment by catalup** [comment]
> If you did reset configuration, then I may generate parameters for your VIN and auto configuration macro for pyren. I need your VIN and screenshot of ID screen from pyren. Also I have to notice that there are complaints that pyren can't write long commands to r-link devices. It is related to timeout parameters. It is probably will require to tune a code a bit for implementing auto-configuration macro on rlink1.

@shrlnm could you please help me too? I am in the same situation. What screenshot is needed to send beside VIN? Thank you in advance!

**Comment by shrlnm** [comment]
@catalup VIN may be  enough

**Comment by catalup** [comment]
> @catalup VIN may be enough

@shrlnm the VIN is:

VF1BT17J650107677

Thank you a lot for help!

**Comment by shrlnm** [comment]
@catalup 
[link removed]

**Comment by catalup** [comment]
> @catalup [link removed]

@shrlnm Could you please tell me more details? I don't know really what to do with the code lines that you've sent me. I will look a bit, but I'm not sure. 

Thank you a lot!

**Comment by shrlnm** [comment]
@catalup I sent you pyren macro for repair your r-link.
Here is how to install the pyren
https://gitlab.com/py_ren/pyren/-/tree/pyren3
and wiki how to use a macro
https://gitlab.com/py_ren/pyren/-/wikis/how%20to%20rlink2%20black%20screen

**Comment by catalup** [comment]
I succeeded using the instructions from this video:

https://youtu.be/h5IOZ-skiD8

Thank you a lot for your help! 
Have a beautiful week!

**Comment by StepanenkoNadejda** [comment]
> > @catalupВИН может быть достаточно
> 
> @shrlnmВИН это:
> 
> VF1BT17J650107677
> 
> Большое спасибо за помощь!

Hello. Please could you help me?) I have a problem with my RLink " ...  unconfigured device please add eol settings ". VIN: VF1BZS00E52144364. PLEASE)

**Comment by StepanenkoNadejda** [comment]
Hello, could someone help me) I have the same problem with my RLink " ...unconfigured device please add eol settings " VIN: VF1BZS00E52144364. PLEASE)

**Comment by andrerochadev92** [comment]
> @catalup VIN may be enough

Can you help me too?
I have the same problem.
VIN 8A1LZV009HL699639

**Comment by felixcastrofuentes** [comment]
@StepanenkoNadejda / @andrerochadev92 can you fix it?

**Comment by vvvvitaliu** [comment]
> ah right thanks. I have repaired my rlink1 in my renault zoe once when it had EOL triangle before with DDT4ALL by filling in settings. Its a bit difficult to explain , but if you have a elm327 bluetooth OBD 2 and the specially wired cable to read the rlink through the OBD2 port and a windows laptop and can get an internet signal on your laptop inside your car I can log on over the internet and have a look remotely from my end to see what settings are filled in and see if i can see other settings missing if you like, if you want me to just let me know.

help me please restore r-link 1.

**Comment by iliapeev** [comment]
@catalup 
Can you help me too?
I have the same problem.
vin:VF1BT17J652378561

**Comment by shrlnm** [comment]
@iliapeev 
> vin:VF1BT17J652378561

--Link removed--

**Comment by iliapeev** [comment]
@shrlnm  thanks.it didn't work. only the logo R-link appeared  
and now only a triangle appeared with the inscription : radio not configured

**Comment by AlMeu** [comment]
The unit which have hardware problems you can't solve via software.

**Comment by shrlnm** [comment]
@AlMeu Hi. Do you think it's hardware problem? I've never had RL1 so I not too aware with it. What do you think if I make a macro for radio module Radio_2012_V2.4_20130604T134716.xml too? Can it help?

**Comment by iliapeev** [comment]
> Устройството, което има хардуерни проблеми, които не можете да разрешите чрез софтуер.

I don't think it's hardware. It broke when deleting errors.

**Comment by shrlnm** [comment]
@iliapeev is your ELM able to connect to multimedia CAN?

**Comment by iliapeev** [comment]
> @iliapeevвашият ELM може ли да се свърже с мултимедиен CAN?

yes.can

**Comment by shrlnm** [comment]
@iliapeev 
--link removed--
try to implement and save the log please.

**Comment by iliapeev** [comment]
> 

thanks, i'll try now

**Comment by iliapeev** [comment]
thanks a lot.it worked.now i just need to find the pin code

**Comment by shrlnm** [comment]
Ok. Perfect.

**Comment by AlMeu** [comment]
@iliapeev 
R-Link 1 doesn't have pin code.
You are sure that you speak about R-Link 1? Maybe you speak about radio,

@shrlnm 
Many times with R-Link 1 when it get in fault you can't even connect to it.

**Comment by iliapeev** [comment]
> @iliapeev R-Link 1 няма пин код. Сигурни ли сте, че говорите за R-Link 1? Може би говориш за радио,
> 
> @shrlnm Много пъти с R-Link 1, когато се повреди, вие дори не можете да се свържете с него.

yes for r-link 1 .first time it ask me for pin .already working

**Comment by poupz13** [comment]
HI
For the same problem, can you help me please.
Vin : ......

thanks a lot

**Comment by AndyIreland** [comment]
> @iliapeev R-Link 1 doesn't have pin code. You are sure that you speak about R-Link 1? Maybe you speak about radio,
> 
>
pretty sure my rlink1 in my Zoe EV car has a security PIN code - had to enter it once when I disconnected battery overnight and when I fixed the EOL message on mine

Is written in back of rlink instruction booklet on mine and also in the 'My Renault' phone app

**Comment by AlMeu** [comment]
@AndyIreland disconnect radio only from connectors and you will see that requests pin code when you connect back.
If you have a screen where you insert it is not by force always for that. You know something, but you don't know how is really working. 
To be a user, doesn't mean you are an engineer also.

**Comment by AndyIreland** [comment]
> @AndyIreland disconnect radio only from connectors and you will see that requests pin code when you connect back. If you have a screen where you insert it is not by force always for that. You know something, but you don't know how is really working. To be a user, doesn't mean you are an engineer also.

Thank you for that info - but all I am saying is that when I disconnected my battery and when I fixed my EOL message when I restarted it I am pretty sure I had to put a PIN in on the rlink touch screen to get it working

**Comment by shrlnm** [comment]
@poupz13 
> Vin : VF13FL00253820445

Try t.me/pyren_bot
then command 
/vin YourVINnumber

Macro cmd_10* is for MFD on main CAN   and macro cmd_76* is for radio on multimedia CAN

**Comment by poupz13** [comment]
Thanks a lot !!! You've save me...!

Now i've an another problem, the fonction auto off is broken. I don't remember the parameter Line...

When i open the door the rlink don't shutdown automatically... Anybody knows it ?


....Edit....

Fu*** it was simply my usb charging cable(double usb data+charging) ... Unplugged always is ok....

Unbelievable!

**Comment by Valentino2891** [comment]
Can someone help me, I can't figure out whether to use the cmd_10 and cmd_76 macros on the can bus or can2?

**Comment by shrlnm** [comment]
> I can't figure out whether to use the cmd_10 and cmd_76 macros on the can bus or can2?

It may depends on platform of car. Look at the name of macro generated by pyren_bot. If it contains _can_ then the macro for a main CAN bus. If the filename contains _can2_ then the macro for multimedia CAN bus.

Anyway, nothing wrong will happen if you'll try to implement it on wrong CAN bus. It just won't connect to the module.

**Comment by Valentino2891** [comment]
> 

![Screenshot_20230410-085529_Gallery](https://user-images.githubusercontent.com/130336232/230836247-a0d23af5-b40a-4d85-b2e7-a3d90e3c5025.jpg)
Дякую

**Comment by AndyIreland** [comment]
CAN2 (and CAN-HI)  is the rlink radio CAN

**Comment by Valentino2891** [comment]
Rlink earned after applying the cmd_10 script, went to reboot and started, everything works!
 Thank you very much…

**Comment by LemurMort** [comment]
Hello!
Is it to late to join here? :)
I'm also getting a triangle and a message 'Radio not configured' after reset configuration.

I opened a new thread not being aware this one's opened: https://github.com/cedricp/ddt4all/issues/815

I only have DDT4ALL.
Can someone help by explaining the steps I need to take?

Thanks in advance!

**Comment by shrlnm** [comment]
@LemurMort what the problem to install pyren3?  I may suggest only this instruction https://www.drive2.ru/b/643019689764615344/ based on pyren

**Comment by busta6** [comment]
> No, i would just copy the TomTom file to fresh SD card not the map card.
> 
> Try use this tomtom.000 file its 11.346 version https://we.tl/t-GCzQGyQoK3

Hello, could you share the link of this update, it does not appear available on the Renault page, thanks

**Comment by Sankyjdm** [comment]
@shrlnm can you help me please, I have the same problem
Vin: VF1KZ030H54916711

**Comment by shrlnm** [comment]
@Sankyjdm the pyren bot is working now so you may do according this article 
https://www.drive2.ru/b/643019689764615344/

**Comment by Sankyjdm** [comment]
Thank you

**Comment by Assh1358** [comment]
Hi
I have similar problem, my radio is work, but I don't have any sound in speakers. Only an intermittent beeping sound comes from the speakers. I open the radio settings, select the default settings (factory) option, after confirming the return to factory settings, the radio does not react. My car is Renault Fluence Bose version 2015 model
Vin: vf1lzlct1fc288501
Please help me

**Comment by matzel82** [comment]
You can disconnect You battery

**Comment by Assh1358** [comment]
> You can disconnect You battery
I did that but nothing changed

**Comment by AndyIreland** [comment]
> Hi I have similar problem, my radio is work, but I don't have any sound in speakers. Only an intermittent beeping sound comes from the speakers. I open the radio settings, select the default settings (factory) option, after confirming the return to factory settings, the radio does not react. My car is Renault Fluence Bose version 2015 model Vin: vf1lzlct1fc288501 Please help me

@Assh1358 - 
Rlink1 is it? - when it was beeping i reckon all you had to do was press the home button 5 times to restart the rlink and then possibly put in the radio unlock code. 

So what are you left with now at present on your screen , is it triangle and EOL message or is screen blank now?

what was you originally try to do? - add android auto to your rlink1 was it?

**Reply by AlMeu** [reply]
As I know there in R-Link 1 is not unlock code, but perhaps is something new.
